app/scripts/controllers/otherResources.js

```diff
@@ -70,6 +70,7 @@ angular.module('openshiftConsole')
       if (!selected) {
         return;
       }
+      $scope.selectedResource = APIService.kindToResource(selected.kind);
```

Comment:
I have and it worked. I dont think we need to specify the apiGroup since `selfsubjectrulesreviews` will return all the resources no matter what group they are in. At least thats what I've saw.
@liggitt is that right ?